cms/blocks.py

```diff
@@ -9,7 +9,28 @@
 from wagtail.images.blocks import ImageChooserBlock
 
 
-class LearningTechniqueBlock(blocks.StructBlock):
+class BlockAPIRepresentationMixin:
+    def get_api_representation(self, value, context=None):
+        """
+        Add the image title and URL to the API representation.
+        """
+        api_representation = super().get_api_representation(value, context)
+        image = value["image"]
+        if image:
+            api_representation.update(
+                {
+                    "image": {
+                        "title": image.title,
+                        "url": image.file.url,
+                        "height": image.height,
+                        "width": image.width,
+                    }
```

Comment:
We can use our image serializer here.
```suggestion
                    "image": ImageSerializer(instance=image).data
```